concepticondata/conceptlists/Verheyen-2019-1000.tsv

```diff
@@ -0,0 +1,1001 @@
+Verheyen-2019-1000-30	30	agressief	aggressive			8.3	5.95	5	5.75	5.4	5.6	2.35	9	3	54558	0	985	587	1.28981883620489	0.00026861324054
+Verheyen-2019-1000-31	31	algemeen	public			7.9	3.55	3.9	3.95	5.25	3.25	4.6	8	3	79200	0	10356	728	1.05964056478197	0.000247496850277
+Verheyen-2019-1000-32	32	allerbest	very best			6.6	5.5	3.75	5.6	3.35	4.95	5.85	9	3	98109	0	181	422	0.010152284263959	3.31926615320873E-07
+Verheyen-2019-1000-33	33	amateuristisch	amateurish			11.35	3.7	4.5	2.9	3.55	3.25	3.25	14	5	130438	0	55	37	0.047270383978851	1.02246414248842E-05
+Verheyen-2019-1000-34	34	ambitieus	ambitious			11.35	5.85	4.25	5.85	3.85	4.9	5.35	9	3	48585	0	220	266	0.328046116190696	6.59297375137343E-05
+Verheyen-2019-1000-35	35	Amerikaans	american			7.9	4.7	5.55	5.1	5.5	5.7	3.6	10	4	106132	0	5191	2	NaN	NaN
+Verheyen-2019-1000-36	36	amper	only just			7.95	2.35	2.3	2.3	5.25	2.3	2.8	5	2	53017	3	0	0	0.483282545736047	7.37853340763283E-05
+Verheyen-2019-1000-37	37	angstig	terrified			6.95	4.85	4.4	3.4	4.8	5.85	2.6	7	2	69882	0	1548	261	1.56889437790534	0.00013747944821
+Verheyen-2019-1000-38	38	antiek	ancient			8.7	2.95	5.15	3.5	2.75	5.75	3.6	6	2	58511	0	769	186	2.21800794232623	0.000325190457539
+Verheyen-2019-1000-39	39	apart	apart			7.55	3.85	4.5	3.55	5	3.7	4.05	5	2	22539	2	2091	915	1.98779504941572	0.000274090029692
+Verheyen-2019-1000-40	40	arm	poor	POOR	1674	4.1	3.2	3.9	3.4	5.85	5.9	2.45	3	1	13759	7	3351	4588	10.3463203572458	0.002099230828407
+Verheyen-2019-1000-41	41	armoedig	shabby			8.65	2.7	3.9	2.75	2.8	5.9	2.1	8	3	43555	0	245	65	0.488449694646872	6.85884046188044E-05
+Verheyen-2019-1000-42	42	arrogant	arrogant			9.85	5.05	4.6	4.85	5.05	4.85	1.75	8	3	49224	0	264	543	2.72885998634532	0.000520294969515
+Verheyen-2019-1000-43	43	artistiek	artistic			10.7	4.9	4.05	5.2	3.6	4.45	4.65	9	3	90729	0	721	208	0.687185080740296	8.27864028800296E-05
+Verheyen-2019-1000-44	44	asociaal	antisocial			10.3	2.25	4.1	3.3	5.35	4.15	1.8	8	3	33308	0	106	65	1.34788407300923	0.000252638458632
+Verheyen-2019-1000-45	45	asymmetrisch	asymmetric			11.45	3.9	5.2	4.25	3.75	6.05	2.95	12	4	73048	0	40	12	0.012448132780083	4.16535360402665E-07
+Verheyen-2019-1000-46	46	avondlijk	evening			11.9285714285714	3.90909090909091	3.75	4.52941176470588	1.82352941176471	3.9	4.88235294117647	9	3	52515	0	50	1	0.004016064257028	3.1890988530829E-07
+Verheyen-2019-1000-47	47	baldadig	rowdy			12.9333333333333	5.5	3.46666666666667	4.88888888888889	1.70588235294118	3.0625	2.875	8	3	32201	0	74	2	0.033604275655867	6.21548858100851E-06
+Verheyen-2019-1000-48	48	bang	afraid	FEAR (BE AFRAID)	1419	3.95	4.35	4.05	2.75	5.75	5.9	2.3	4	1	32714	14	7921	21119	9.56742495937603	0.001036131709002
+Verheyen-2019-1000-49	49	bar	barren			8.8	3.68421052631579	4.29411764705882	3.05263157894737	3.73684210526316	5.65	3.9	3	1	14841	19	725	98	1.39625465570824	0.000294051185167
+Verheyen-2019-1000-50	50	barbaars	wild	UNTAMED OR WILD	2230	10.95	5.3	4.4	4.6	2.7	4.9	2.15	8	2	51975	0	292	153	0.102223583069015	2.93429636308658E-05
```

Comment:
barbaars is barbaric, not wild, so the concept wild or untamed does not fit completely I think?